editors/tinymce5/fontawesome/plugin.min.js

```diff
@@ -0,0 +1,4781 @@
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Bitbucket",
+			"id": "bitbucket",
+			"unicode": "f171",
+			"created": 3.2,
+			"filter": ["git"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Bitbucket Square",
+			"id": "bitbucket-square",
+			"unicode": "f172",
+			"created": 3.2,
+			"filter": ["git"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Tumblr",
+			"id": "tumblr",
+			"unicode": "f173",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Tumblr Square",
+			"id": "tumblr-square",
+			"unicode": "f174",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Long Arrow Down",
+			"id": "long-arrow-down",
+			"unicode": "f175",
+			"created": 3.2,
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Long Arrow Up",
+			"id": "long-arrow-up",
+			"unicode": "f176",
+			"created": 3.2,
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Long Arrow Left",
+			"id": "long-arrow-left",
+			"unicode": "f177",
+			"created": 3.2,
+			"filter": ["previous", "back"],
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Long Arrow Right",
+			"id": "long-arrow-right",
+			"unicode": "f178",
+			"created": 3.2,
+			"categories": ["Directional Icons"]
+		}, {
+			"name": "Apple",
+			"id": "apple",
+			"unicode": "f179",
+			"created": 3.2,
+			"filter": ["osx", "food"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Windows",
+			"id": "windows",
+			"unicode": "f17a",
+			"created": 3.2,
+			"filter": ["microsoft"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Android",
+			"id": "android",
+			"unicode": "f17b",
+			"created": 3.2,
+			"filter": ["robot"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Linux",
+			"id": "linux",
+			"unicode": "f17c",
+			"created": 3.2,
+			"filter": ["tux"],
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Dribbble",
+			"id": "dribbble",
+			"unicode": "f17d",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Skype",
+			"id": "skype",
+			"unicode": "f17e",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Foursquare",
+			"id": "foursquare",
+			"unicode": "f180",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Trello",
+			"id": "trello",
+			"unicode": "f181",
+			"created": 3.2,
+			"categories": ["Brand Icons"]
+		}, {
+			"name": "Female",
+			"id": "female",
+			"unicode": "f182",
+			"created": 3.2,
+			"filter": ["woman", "user", "person", "profile"],
+			"categories": ["Web Application Icons"]
+		}, {
+			"name": "Male",
+			"id": "male",
+			"unicode": "f183",
+			"created": 3.2,
+			"filter": ["man", "user", "person", "profile"],
+			"categories": ["Web Application Icons"]
+		}, {
+			"name": "Gratipay (Gittip)",
+			"id": "gratipay",
+			"unicode": "f184",
+			"created": 3.2,
+			"aliases": ["gittip"],
+			"filter": ["heart", "like", "favorite", "love"],
+			"categories": ["Brand Icons"]
+
```

Comment:
@cmb69: Beim Öffnen des Fensters stimmt irgendwas mit der Positionierung nicht. Wenn während den Öffnens die Accordions geschlossen sind, funktioniert es. Vielleicht siehst Du, woran es liegt.

Reply:
Hm, bei mir sind die Accordions beim Öffnen wohl immer geschlossen. :?

Reply:
Ja eben. Das Öffnen des ersten Accordions habe ich im code auskommentiert. Wenn Du diese wieder aktivierst, siehst Du das Problem mit dem focus im Fenster/oder auch falsche Fensterhöhe.

Reply:
Ah, ja (passiert wohl im FF, aber nicht im Chrome). Da scheint allgemein etwas nicht in Ordnung: ist der Dialog zu klein, und man öffnet z.B. „Web Application“, dann verschwindet der Schließen-Schalter. Setzt man das `overflow:auto` für `.tox-dialog__content-j` statt für `.tox-dialog__body-content`, dann passt es besser; allerdings ruckelt es dann. Ich frage mich, ob man nicht ganz auf das Accordion verzichten sollte – einen besonderen Mehrwert scheint dieses hier nicht zu haben.

Reply:
Uff, das war eine Frickelei. Ich habe es nun so gelöst, dass `.mce-fontawesome-panel` nicht an den vom TinyMCE5 erzeugten Dialog gesetzt wird, sondern ein Wrapper div mit dieser Klasse eingeführt. Da scheint das nun alles wesentlich besser zu funktionieren.